docs/release-notes/2024-06-13-new-logistics-api-endpoints-to-update-inventory-lead-time-and-quantity.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "New Logistics API endpoints to update inventory lead time and quantity"
+slug: "2024-06-13-new-logistics-api-endpoints-to-update-inventory-lead-time-and-quantity"
+hidden: false
+type: "added"
+createdAt: "2024-06-14T12:04:00.219Z"
+excerpt: "Manage SKU quantity and lead time in separate endpoints."
+---
+
+In order to facilitate your [inventory](https://help.vtex.com/en/tutorial/inventory-management--tutorials_139) management, we have launched new endpoints in our **Logistics API**. In addition to the previous [Update inventory by SKU and warehouse](https://developers.vtex.com/docs/api-reference/logistics-api#put-/api/logistics/pvt/inventory/skus/-skuId-/warehouses/-warehouseId-), you can now, if desired, use an endpoint to configure only lead time and another just to update SKU quantity.
```

Comment:
Add the term "management" within the link, as they are together in the title of the linked article.
[inventory management](https://help.vtex.com/en/tutorial/inventory-management--tutorials_139)